src/sickle.ts

```diff
@@ -238,8 +252,8 @@ function last<T>(elems: T[]): T {
   return elems.length ? elems[elems.length - 1] : null;
 }
 
-export function annotate(file: ts.SourceFile): string {
-  return new Annotator().annotate(file);
+export function annotate(file: ts.SourceFile, options?: SickleOptions): string {
+  return new Annotator(options || defaultOptions).annotate(file);
```

Comment:
So if you keep adding defaultOptions and values, this currently either takes the defaults, or the given options, but doesn't individually override, which is the behaviour I'd expect. Does that make sense?